internal/resources/packages/packageslist/cluster_scope.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package listpackages
+
+import (
+	packageclustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/package/cluster"
+
```

Comment:
Why is this extra line needed?

Please check all imports.

Reply:
reabsed it